src/styles/blocks/calender.scss

```diff
@@ -0,0 +1,56 @@
+.calender {
+  width: $width-calc;
+  display: flex;
+  flex-direction: row;
+
+  flex-wrap: wrap;
+  gap: $box-gap;
+  padding: $padding;
+
+  &__box {
+    background-color: $calender-color;
+    height: $box-size;
+    line-height: $box-size;
+    display: flex;
+
+    justify-content: center;
+    align-items: center;
+    color: black;
+    font-family: Arial, Helvetica, sans-serif;
+
+    font-size: 30px;
+    text-align: center;
+    flex-basis: $box-size;
+    border: 1px solid black;
+
+    transition: background-color, transform 500ms ease-in-out;
+
+    &:hover {
+      cursor: pointer;
+      background: $block-hover-bg;
+      transform: translateY(-20px);
+    }
+  }
+
+  @each $day, $i in $week-days {
+    &--start-day_#{$day} {
+      .calender__box:first-child {
+        margin-left: ($box-size + 2 + $box-gap) * $i ;
```

Comment:
What if we increase the size of border this formula also needs to change. You may use box-sizing property to control this.